.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Build
+        run: |
+          chmod +x gradlew
```

Comment:
Why not to commit `gradlew` with executable permissions right into the repository?

Reply:
> Why not to commit `gradlew` with executable permissions right into the repository?

fixed